packages/main/src/plugin/certificates.ts

```diff
@@ -81,6 +81,8 @@ export class Certificates {
       if (import.meta.env.PROD) {
         const rootExePath = path.join(process.resourcesPath, 'win-ca', 'roots.exe');
         wincaAPI.exe(rootExePath);
+      } else {
+        wincaAPI.exe(path.resolve(__dirname, '..', '..', '..', 'node_modules', 'win-ca', 'lib', 'roots.exe'));
```

Comment:
```suggestion
        wincaAPI.exe(require.resolve('win-ca/lib/roots.exe'));
```

Reply:
verified on windows and it is the better way to resolve path to roots.exe.